contracts/modules/claims/Claims.sol

```diff
@@ -173,38 +173,29 @@ contract Claims is Iupgradable {
    * @param coverId Cover Id.
    */
   function submitClaim(uint coverId) public {
-    address qadd = qd.getCoverMemberAddress(coverId);
-    require(qadd == msg.sender);
-    uint8 cStatus;
-    (, cStatus,,,) = qd.getCoverDetailsByCoverID2(coverId);
-    require(cStatus != uint8(QuotationData.CoverStatus.ClaimSubmitted), "Claim already submitted");
-    require(cStatus != uint8(QuotationData.CoverStatus.CoverExpired), "Cover already expired");
-    if (ms.isPause() == false) {
-      _addClaim(coverId, now, qadd);
-    } else {
-      cd.setClaimAtEmergencyPause(coverId, now, false);
-      qd.changeCoverStatusNo(coverId, uint8(QuotationData.CoverStatus.Requested));
-    }
+
+    require(!ms.isPause(), "Claims: System is paused");
+
+    address coverOwner = qd.getCoverMemberAddress(coverId);
+    require(coverOwner == msg.sender, 'Claims: Not cover owner');
+
+    uint coverStatus = qd.getCoverStatusNo(coverId);
+    require(coverStatus != uint8(QuotationData.CoverStatus.ClaimSubmitted), "Claim already submitted");
```

Comment:
Needs:
1. Claim count per cover
2. Status != ClaimAccepted
3. Need a way to know accepted status for expired covers